webpack.config.js

```diff
@@ -0,0 +1,73 @@
+'use strict'
+
+var path = require('path')
+var webpack = require('webpack')
+var HtmlWebpackPlugin = require('html-webpack-plugin')
+
+module.exports = {
+  devtool: 'eval-source-map',
+  target: 'electron',
+  entry: [
+    'webpack-dev-server/client?http://localhost:3000',
+    'webpack/hot/only-dev-server',
+    'react-hot-loader/patch',
+    path.join(__dirname, 'app/index.js')
+  ],
+  externals : {
```

Comment:
@coranos can you please add the externals config option to the webpack dev config as well. This way we can also run 'yarn run assets-dev'. Thanks! and great work on this today. Tomorrow, I will look at getting all the tests working and will probably have some comments on a few of the functions you changed as well.